index.js

```diff
@@ -1,21 +1,26 @@
 /* eslint-env node */
 'use strict'
 
+const mergeTrees = require('broccoli-merge-trees');
+const writeFile = require('broccoli-file-creator');
+const version = require('./package.json').version;
 const VersionChecker = require('ember-cli-version-checker');
 
+const minEmberVersion = '3.0.0-beta.3';
+
 module.exports = {
   name: 'ember-2-legacy',
 
   init() {
     this._super && this._super.init.apply(this, arguments);
 
-    let checker = new VersionChecker(this);
+    let checker = new VersionChecker(this.parent);
```

Comment:
@rwjblue I tested this in a fresh app and it wasn't grabbing the correct version (only grabbing its own version). Just want to verify that this is the best way to grab the parent apps version

Reply:
Ya, this change seems correct.